### ISSUE TITLE
Downloaded pre-compiled, keeps telling me "you need to put torches in your first and last inventory slot"

### ISSUE THREAD

**Issue by graviaDaemon** [RussDev7/CoreKeepersWorkshop#8]
So I've downloaded this program pre-compiled from the releases tab.
Extracted it to my desired location.
Then ran the program while Core Keeper was running, and then put a single torch in my first hotbar slot (number 1) and in the last slot of my inventory (ignoring any added inventory slots from bags) So the second row of inventory (3rd if you count the hotbar) all the way on the right.
It pops up, telling me "you need to put torches in your first and last slot, ignore additional rows..."
So I change things up, I empty out my inventory, empty out my hotbar.
Then I try again.
1 single torch in my first hotbar slot, and in the last of the inventory (now no bags equipped)
still nothing
So I try the first inventory slot, and not the hotbar slot
still nothing
I've tried every possible combination, unequipped everything I had and emptied out my inventory just for the torches...

What am I doing wrong then?
It keeps telling me the same thing.
I've re-downloaded the thing a couple of times now..

**Comment by RussDev7** [comment]
Hello, are you using r2modman or other mods?

**Comment by graviaDaemon** [comment]
> Hello, are you using r2modman or other mods?

Yes, I am. Is that creating conflicts?

**Comment by RussDev7** [comment]
> > Hello, are you using r2modman or other mods?
> 
> Yes, I am. Is that creating conflicts?

Yes this seems to be what's cold be causing some conflicts. This mod had recently updated its headers for inventory loading, I'll look into this problem.

**Comment by RussDev7** [comment]
> > > Hello, are you using r2modman or other mods?
> > 
> > 
> > Yes, I am. Is that creating conflicts?
> 
> Yes this seems to be what's cold be causing some conflicts. This mod had recently updated its headers for inventory loading, I'll look into this problem.

So to follow up on this, I have tested this vigorously without fail on r2modman / modded. I would suggest running the application as administrator and following all the requirements on the front page.
